### ISSUE TITLE
[BUG] Can't Find Module Webhook

### ISSUE THREAD

**Issue by BlackYuzia** [matthew1232/discord-webhook-node#10]
### Description

I was create some script.
When I run it on my **Windows 7** with **node 11.0.0**, all work successfully.
But when I try run it on ~~Kruzuntu~~ **Ubuntu 18.04** with **node 11.0.0**, I was get error about can't find module.

What the type of bug this is kinda be?

### Examples

![image](https://user-images.githubusercontent.com/25010528/78396146-7b0c5880-75f7-11ea-92aa-9f45ac7c2ed4.png)
```

Error: Cannot find module './classes/Webhook'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:587:15)
    at Function.Module._load (internal/modules/cjs/loader.js:513:25)
    at Module.require (internal/modules/cjs/loader.js:643:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at Object.<anonymous> (/home/ubuntu/bots/qiwi-box/node_modules/discord-webhook-node/src/index.js:1:79)
    at Module._compile (internal/modules/cjs/loader.js:707:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:718:10)
    at Module.load (internal/modules/cjs/loader.js:605:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:544:12)
    at Function.Module._load (internal/modules/cjs/loader.js:536:3)

```

**Comment by BlackYuzia** [comment]
### Fix

 **Rename** `./classes/webhook` to `./classes/webhook`

On Linux, `Webhook` and `webhook` **this a two different files**.
On **Windows**, `Webhook` and `webhook` **this a the same files**.

PS: I try send PR later

**Comment by EramsorGR** [comment]
The fix works, just need to run 
`mv node_modules/discord-webhook-node/src/classes/webhook.js node_modules/discord-webhook-node/src/classes/Webhook.js`
and
`mv node_modules/discord-webhook-node/src/classes/messageBuilder.js node_modules/discord-webhook-node/src/classes/MessageBuilder.js`

**Comment by matthew1232** [comment]
Sorry about that that, small error on my part. I updated the file names and
published to 1.0.7 so you can reinstall the module and it will work.

**Comment by matthew1232** [comment]
Fixed in 1.1.0, Github didn't recognize when I changed the file name to uppercase, causing it to break on case sensitive file systems. Sorry about the inconvenience.

**Comment by chunkun** [comment]
I found a similar problem in version 1.1.0.

I fixed the source code to solve the problem.

**node_modules/discord-webhook-node/src/classes/webhook.js**
`require('./messageBuilder') -> require('./MessageBuilder')`

**node_modules/discord-webhook-node/src/index.js**
`require('./classes/messageBuilder') -> require('./classes/MessageBuilder');`
